website/templates/index.html

```diff
@@ -46,14 +46,14 @@ <h2>Energy cells explained</h2>
             <p>The fundamental idea of the cellular approach is to balance energy consumption and energy production at the smallest level possible</p>
```

Comment:
I suggest this change here as it seems to be more in line with the "selling points" descibed lower for open_plan.
(+It seems that not everyone agrees on this balancing of consumption and generation in an energy cell)

```suggestion
            <p>The fundamental idea of the cellular approach is to optimize energy system planning at all possible levels, from the smallest energy cell to the largest one.</p>
```

Reply:
I agree to your suggestion! Shall I commit it accordingly by clickling on commit suggestion? --> I am not sure about the correct GitHub Etiqutte ;)

Reply:
yes :)

Reply:
Then you can click "merge" since Andra approved your pull request :)